codec/src/main/java/io/netty/handler/codec/LengthFieldBasedFrameDecoder.java

```diff
@@ -354,7 +338,8 @@ protected final void decode(ChannelHandlerContext ctx, ByteBuf in, List<Object>
      * @return  frame           the {@link ByteBuf} which represent the frame or {@code null} if no frame could
      *                          be created.
      */
-    protected Object decode(ChannelHandlerContext ctx, ByteBuf in) throws Exception {
+    protected Object decode(@SuppressWarnings({ "unused", "squid:S1172" }) ChannelHandlerContext ctx, ByteBuf in)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS1161)